service/src/main/java/com/commercetools/pspadapter/payone/notification/common/AppointedNotificationProcessor.java

```diff
@@ -106,15 +116,13 @@ private static void addPaymentUpdatesForNewAuthorizationTransaction(
                 .build()));
     }
 
-    private static void addPaymentUpdatesForNewChargeTransaction(
-            final Notification notification,
-            final ImmutableList.Builder<UpdateAction<Payment>> listBuilder) {
+    private static AddTransaction addChargePendingTransaction(final Notification notification) {
```

Comment:
> more like createChargePendingTransaction since it's not "adding" anymore ;)

`AddTransaction` is a name of update action, so name of the function more or less relevant.

Reply:
>Also is the @Nullable annotations missing? what if the notification is null, NPE?

like in most of the sources of the project, unfortunately (and we know, `@Nullable` annotation won't save us from NPE :) ). I'm consistently improving the sources with such annotations, but usually only those methods i re-factor.
f5b54a91

Reply:
> AddTransaction is a name of update action, so name of the function more or less relevant.

Sorry, `createChargePendingTransaction` is a wrong suggestion I meant sth in the lines of `createUpdateAction` instead of `addTransaction` which the method doesn't actually "add" but it actually creates an updates action called "add.."